packages/scheduler/src/queueMicrotask.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+function queueMicrotaskPolyfill(callback: boolean => void) {
+  if (typeof queueMicrotask !== 'undefined') {
+    queueMicrotask(callback);
+  } else if (typeof Promise !== 'undefined') {
+    Promise.resolve(null)
+      .then(callback)
+      .catch(e => {
```

Comment:
This function can be hoised to be the same instance instead of a closure.

Reply:
Done

Reply:
If you can, please double check that's what you meant.